_tools/cite-guide.md

```diff
@@ -0,0 +1,459 @@
+
+1. Ensure proper functionality of system changes prior to production deployment, and
+2. Identify and resolve technical issues across FPKI Partners.
+
+The FPKI Community can use CITE to evaluate PKI or application changes in a test environment that mimics the production FPKI hierarchy and test potential interoperability issues before those changes are deployed to the FPKI. <br>
+
+## Document Scope
+
+This document provides guidelines and specifications for an FPKI Partner to participate in CITE. CITE Participants refer to an FPKI Partner establishing a test PKI certified or cross-certified with the Test Common Policy or Test Bridge CA. This document shall not define how CITE Participants conduct testing.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+Modifications to this document and any waivers are controlled by the FPKI Technical Working Group. <br>
+
+## Testing Use Cases
+
+The main purpose of CITE is interoperability and infrastructure testing of PKI components and Relying Party applications. Additional types of testing may be identified and conducted as necessary and to the extent supported by CITE Participants. CITE should not be used for system stress testing. Infrastructure testing ensures that upgrades, patches, policy changes, new products, and any other changes to the production FPKI do not adversely affect interoperability.
+
+Relying Party application testing ensures that application modules operate as intended. In addition, application testing ensures that the system performs as expected and can properly process transactions that rely on FPKI certificates.
+
+Some examples of testing conducted in CITE include:
+1. Interoperability testing between cross-certified Certification Authorities (CAs);
+2. Transition testing to new algorithms (e.g., SHA-2, ECC);
+3. PIV and PIV-I card interoperability testing;
+4. Repository access testing when using content delivery networks, load balancers, or other networking configurations; and
+5. Path discovery and/or validation testing for an application
+
+When testing is successful in CITE, assurance is gained that the proposed change(s) will operate in the production FPKI as intended. When tests fail in the CITE, issues are identified and addressed without production FPKI impact. <br>
+
+## Terms and Conditions

```

Comment:
This isn't a legal agreement - probably should  use other words than "Terms and Conditions"

Reply:
Done.